Extensions/tf2_reblogs.js

```diff
@@ -0,0 +1,34 @@
+
+	run: function() {
+		this.running = true;
+		XKit.post_listener.add("tf2_reblogs", XKit.extensions.tf2_reblogs.change_icon);
+		XKit.extensions.tf2_reblogs.change_icon();
+	},
+
+	change_icon: function() {
+		$(".reblog_icon").each(function() {
+			var iconurl = XKit.extensions.tf2_reblogs.icons[Math.floor(Math.random() * XKit.extensions.tf2_reblogs.icons.length)];
+			$(this).replaceWith('<img class = "tf2_icon" src="' + iconurl + '" height = "20px" style = "vertical-align:top;margin:0 5px">');
```

Comment:
the HTML attributes shouldn't have spaces - should be `class="tf2_icon"` for example. conversely, that inline CSS should be spaced for readability.